lib/node_modules/@stdlib/math/base/special/fibonacci-index/benchmark/benchmark.js

```diff
@@ -37,14 +38,16 @@ bench( pkg, function benchmark( b ) {
 	var y;
 	var i;
 
-	FN = new Array( 76 );
+	FN = zeros( 76 );
 	for ( i = 3; i < 79; i++ ) {
 		FN[ i ] = fibonacci( i );
 	}
+
+	x = floor( uniform( 100, 3.0, 78.0 ) );
```

Comment:
This doesn't work the way you are expecting it to, as `floor` only accepts scalars. Instead, rather than `uniform`, use `discrete-uniform`.